docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "3.8"
+services:
+
+  scheduler:
+    image: ocurrent/ocluster-scheduler:live
+    user: "1001:1002"
+    command:
+      - --secrets-dir=/local.data/capnp-secrets
+      - --capnp-secret-key-file=/local.data/capnp-secrets/key.cap
+      - --capnp-listen-address=tcp:0.0.0.0:9000
+      - --capnp-public-address=tcp:scheduler:9000
+      - --pools=solver
+      - --state-dir=/local.data/ocluster-scheduler
+    init: true
+    ports:
+      - "9000:9000"
+    volumes:
+      - 'local.data:/local.data'
+
+  solver-worker:
+    build:
+      dockerfile: Dockerfile
+      context: .
+    user: "1001:1002"
+    command:
+      - --connect=/local.data/capnp-secrets/pool-solver.cap
+      - --name=solver-1
+      - --capacity=10
+      - --state-dir=/local.data/ocluster
+      - --verbose
+    init: true
+    privileged: true            # required for the Docker in Docker container to work
```

Comment:
This shouldn't be necessary since this worker doesn't use docker, unlike the ocluster-worker. Please remove.